modules/utxo-lib/test/inscription.ts

```diff
@@ -0,0 +1,25 @@
+import * as assert from 'assert';
+import { inscriptions } from '../src';
+
+describe('inscriptions', () => {
+  describe('Inscription Address', () => {
+    const pubKey = 'af455f4989d122e9185f8c351dbaecd13adca3eef8a9d38ef8ffed6867e342e3';
+    const pubKeyBuffer = Buffer.from(pubKey, 'hex');
+
+    it('should generate an inscription address', () => {
+      const contentType = 'text/plain';
+      const inscriptionData = new Uint8Array(Buffer.from('Never Gonna Give You Up', 'ascii'));
+
+      const address = inscriptions.makeInscriptionAddress(pubKeyBuffer, inscriptionData, contentType);
+      assert(address);
```

Comment:
The `assert` is not really doing anything - the TS signature already guarantees that there will be a return value.

Let's instead make sure it matches some value

```suggestion
      const script = inscriptions.createOutputScriptForInscription(pubKeyBuffer, contentType, inscription data);
      assert.strictEquals(script.toString('hex'), 'deadbeef');
      assert.strictEquals(fromOutputScript(script, networks.testnet), 'whatever');
```